grid2op/utils/l2rpn_idf_2023_scores.py

```diff
@@ -7,8 +7,9 @@
 # This file is part of Grid2Op, Grid2Op a testbed platform to model sequential decision making in power systems.
 
 from grid2op.utils.l2rpn_2020_scores import ScoreL2RPN2020
-from grid2op.Reward import L2RPNSandBoxScore, _NewRenewableSourcesUsageScore, _AssistantConfidenceScore, _AssistantCostScore
+from grid2op.Reward import L2RPNSandBoxScore, _NewRenewableSourcesUsageScore, _AlertTrustScore  # , _AlertCostScore
```

Comment:
This has been commented, we should maybe juste remove it (we no longer have two components in the assistant score).